.circleci/config.yml

```diff
@@ -54,6 +54,60 @@ jobs:
             cargo test --lib --all-features
       # No need to save this in cache
 
+  tests_ffi:
+    docker:
+      - image: zondax/rust-ci:latest
+    steps:
+      - checkout
+      - restore-sccache-cache
+      - run:
+          name: Build, test and run everything related to FFI
+          command: |
```

Comment:
IMO, each language should have its own job in a separated file, say `ffi.yml`.

Reply:
AFAIK there is yet no support for multiple separate configuration files...

Reply:
AFAIK, Circleci has not yet support for multiple files...

I would suggest the following:
- move `sudo apt to its own `run` called dependencies
- create a small shell script `test_LANG` for each LANG is c, cpp, java, etc.
- create separate runs for each: `run: ./test_cpp` .. etc.
That way CI dashboard is easier to understand, and eventually the scripts can be run locally too
- create a `run: ./test_LANG` so they are not 
- split this in `run` statements so it is possible to quickly see which language failed in CI
- move the

Reply:
Maybe one statement is missing? the `* move the ...`